Windows Java 11 native build failures after Smallrye JWT bump to 3.6.0

### Describe the bug

Once https://github.com/quarkusio/quarkus/pull/30057 got merge, our [Beefy Scenarios](https://github.com/quarkus-qe/beefy-scenarios) daily build fails in native mode. Related to EdDsa by Jose4j.

Full logs here https://github.com/quarkus-qe/beefy-scenarios/actions/workflows/daily.yaml

### Expected behavior

Native build succeeds.

### Actual behavior

Native build fails with following exception message:
```bash
2022-12-31T23:39:39.5197666Z ========================================================================================================================
2022-12-31T23:39:39.5202238Z GraalVM Native Image: Generating '003-quarkus-many-extensions-1.0.0-SNAPSHOT-runner' (executable)...
2022-12-31T23:39:39.5206246Z ========================================================================================================================
2022-12-31T23:40:09.7449661Z [1/7] Initializing...                                                                                   (58.4s @ 0.51GB)
2022-12-31T23:40:09.8407970Z  Version info: 'GraalVM 22.3.0 Java 11 CE'
2022-12-31T23:40:09.8422640Z  Java version info: '11.0.17+8-jvmci-22.3-b08'
2022-12-31T23:40:09.9085952Z  C compiler: cl.exe (microsoft, x64, 19.34.31937)
2022-12-31T23:40:09.9088451Z  Garbage collector: Serial GC
2022-12-31T23:40:09.9598928Z  10 user-specific feature(s)
2022-12-31T23:40:09.9604542Z  - io.quarkus.caffeine.runtime.graal.CacheConstructorsFeature
2022-12-31T23:40:10.1079445Z  - io.quarkus.hibernate.orm.runtime.graal.DisableLoggingFeature: Disables INFO logging during the analysis phase for the [org.hibernate.Version, org.hibernate.annotations.common.Version, org.hibernate.dialect.Dialect] categories
2022-12-31T23:40:10.1102966Z  - io.quarkus.hibernate.validator.runtime.DisableLoggingFeature: Disables INFO logging during the analysis phase for the [org.hibernate.validator.internal.util.Version] categories
2022-12-31T23:40:10.1107781Z  - io.quarkus.jdbc.postgresql.runtime.graal.SQLXMLFeature
2022-12-31T23:40:10.1109067Z  - io.quarkus.runner.Feature: Auto-generated class by Quarkus from the existing extensions
2022-12-31T23:40:10.1153254Z  - io.quarkus.runtime.graal.DisableLoggingFeature: Disables INFO logging during the analysis phase for the [org.jboss.threads] categories
2022-12-31T23:40:10.1154546Z  - io.quarkus.runtime.graal.ResourcesFeature: Register each line in META-INF/quarkus-native-resources.txt as a resource on Substrate VM
2022-12-31T23:40:10.1155853Z  - io.quarkus.websockets.client.runtime.DisableLoggingFeature: Disables INFO logging during the analysis phase for the [io.undertow.websockets] categories
2022-12-31T23:40:10.1157112Z  - org.hibernate.graalvm.internal.GraalVMStaticFeature: Hibernate ORM's static reflection registrations for GraalVM
2022-12-31T23:40:10.1158243Z  - org.hibernate.graalvm.internal.QueryParsingSupport: Hibernate ORM's support for HQL Parser in GraalVM
2022-12-31T23:40:15.0139218Z WARNING: An illegal reflective access operation has occurred
2022-12-31T23:40:15.3920613Z WARNING: Illegal reflective access by io.netty.util.internal.ReflectionUtil (file:/D:/a/beefy-scenarios/beefy-scenarios/003-quarkus-many-extensions/target/003-quarkus-many-extensions-1.0.0-SNAPSHOT-native-image-source-jar/lib/io.netty.netty-common-4.1.86.Final.jar) to constructor java.nio.DirectByteBuffer(long,int)
2022-12-31T23:40:15.7342704Z WARNING: Please consider reporting this to the maintainers of io.netty.util.internal.ReflectionUtil
2022-12-31T23:40:16.2031920Z WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
2022-12-31T23:40:16.3953966Z WARNING: All illegal access operations will be denied in a future release
2022-12-31T23:41:43.2166223Z [2/7] Performing analysis...  []                                                                        (92.9s @ 1.54GB)
2022-12-31T23:41:43.2596905Z   10,332 (90.70%) of 11,391 classes reachable
2022-12-31T23:41:43.2597573Z   13,836 (59.15%) of 23,392 fields reachable
2022-12-31T23:41:43.2598138Z   42,679 (80.11%) of 53,273 methods reachable
2022-12-31T23:41:43.2641382Z      750 classes,     0 fields, and     0 methods registered for reflection
2022-12-31T23:41:43.2719659Z 2 fatal errors detected:
2022-12-31T23:41:43.2720061Z        2 native libraries: psapi, winhttp
2022-12-31T23:41:43.2758367Z Fatal error: com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered while parsing org.jose4j.keys.EdDsaKeyUtil.privateKey(byte[], java.lang.String) 
2022-12-31T23:41:43.2758922Z 
2022-12-31T23:41:43.2759342Z Parsing context:
2022-12-31T23:41:43.2765552Z    at org.jose4j.keys.EdDsaKeyUtil.privateKey(EdDsaKeyUtil.java:98)
2022-12-31T23:41:43.2766539Z    at org.jose4j.keys.EdDsaKeyUtil.privateKey(EdDsaKeyUtil.java:21)
2022-12-31T23:41:43.2805311Z    at org.jose4j.jwk.OctetKeyPairJsonWebKey.<init>(OctetKeyPairJsonWebKey.java:89)
2022-12-31T23:41:43.2806053Z    at org.jose4j.jwk.OctetKeyPairJsonWebKey.<init>(OctetKeyPairJsonWebKey.java:63)
2022-12-31T23:41:43.2806451Z 
2022-12-31T23:41:43.2826243Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisError.parsingError(AnalysisError.java:153)
2022-12-31T23:41:43.2827255Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:104)
2022-12-31T23:41:43.2828498Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.ensureFlowsGraphCreated(MethodTypeFlow.java:83)
2022-12-31T23:41:43.2829665Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.getOrCreateMethodFlowsGraph(MethodTypeFlow.java:65)
2022-12-31T23:41:43.2830860Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.typestate.DefaultVirtualInvokeTypeFlow.onObservedUpdate(DefaultVirtualInvokeTypeFlow.java:109)
2022-12-31T23:41:43.2837580Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.TypeFlow.update(TypeFlow.java:562)
2022-12-31T23:41:43.2838460Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis$1.run(PointsToAnalysis.java:488)
2022-12-31T23:41:43.2839405Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.executeCommand(CompletionExecutor.java:193)
2022-12-31T23:41:43.2840425Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.lambda$executeService$0(CompletionExecutor.java:177)
2022-12-31T23:41:43.2847728Z 	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1426)
2022-12-31T23:41:43.2929984Z 	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
2022-12-31T23:41:43.8038018Z 	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
2022-12-31T23:41:43.8039500Z 	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
2022-12-31T23:41:43.8040743Z 	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
2022-12-31T23:41:43.8049931Z 	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
2022-12-31T23:41:43.8639377Z Caused by: org.graalvm.compiler.java.BytecodeParser$BytecodeParserError: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: java.security.spec.EdECPrivateKeySpec. This error is reported at image build time because class org.jose4j.keys.EdDsaKeyUtil is registered for linking at image build time by command line
2022-12-31T23:41:43.8640681Z 	at parsing org.jose4j.keys.EdDsaKeyUtil.privateKey(EdDsaKeyUtil.java:99)
2022-12-31T23:41:43.8641574Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.throwParserError(BytecodeParser.java:2518)
2022-12-31T23:41:43.8650409Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.throwParserError(SharedGraphBuilderPhase.java:110)
2022-12-31T23:41:43.8651702Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3393)
2022-12-31T23:41:43.8652674Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.handleBytecodeBlock(BytecodeParser.java:3345)
2022-12-31T23:41:43.8653555Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBlock(BytecodeParser.java:3190)
2022-12-31T23:41:43.8666191Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.build(BytecodeParser.java:1138)
2022-12-31T23:41:43.8667264Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.buildRootMethod(BytecodeParser.java:1030)
2022-12-31T23:41:43.8668114Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.GraphBuilderPhase$Instance.run(GraphBuilderPhase.java:97)
2022-12-31T23:41:43.8669711Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase.run(SharedGraphBuilderPhase.java:84)
2022-12-31T23:41:43.8678846Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.run(Phase.java:49)
2022-12-31T23:41:43.8679822Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:446)
2022-12-31T23:41:43.8680558Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:42)
2022-12-31T23:41:43.8681448Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:38)
2022-12-31T23:41:43.8682288Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.AnalysisParsedGraph.parseBytecode(AnalysisParsedGraph.java:135)
2022-12-31T23:41:43.8694083Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.ensureGraphParsed(AnalysisMethod.java:685)
2022-12-31T23:41:43.8696754Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.parse(MethodTypeFlowBuilder.java:171)
2022-12-31T23:41:43.8697849Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.apply(MethodTypeFlowBuilder.java:349)
2022-12-31T23:41:43.8698841Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:93)
2022-12-31T23:41:43.8705515Z 	... 13 more
2022-12-31T23:41:43.8706621Z Caused by: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: java.security.spec.EdECPrivateKeySpec. This error is reported at image build time because class org.jose4j.keys.EdDsaKeyUtil is registered for linking at image build time by command line
2022-12-31T23:41:43.8708227Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.reportUnresolvedElement(SharedGraphBuilderPhase.java:333)
2022-12-31T23:41:43.8715202Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.handleUnresolvedType(SharedGraphBuilderPhase.java:288)
2022-12-31T23:41:43.8716589Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.handleUnresolvedNewInstance(SharedGraphBuilderPhase.java:204)
2022-12-31T23:41:43.8717973Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.genNewInstance(BytecodeParser.java:4501)
2022-12-31T23:41:43.8719055Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.genNewInstance(BytecodeParser.java:4494)
2022-12-31T23:41:43.8720005Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBytecode(BytecodeParser.java:5291)
2022-12-31T23:41:43.8728911Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3385)
2022-12-31T23:41:43.8729572Z 	... 28 more
2022-12-31T23:41:43.8730296Z Fatal error: com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered while parsing org.jose4j.keys.EdDsaKeyUtil.publicKey(byte[], java.lang.String) 
2022-12-31T23:41:43.8730995Z Parsing context:
2022-12-31T23:41:43.8735818Z    at org.jose4j.keys.EdDsaKeyUtil.publicKey(EdDsaKeyUtil.java:73)
2022-12-31T23:41:43.8736543Z    at org.jose4j.keys.EdDsaKeyUtil.publicKey(EdDsaKeyUtil.java:21)
2022-12-31T23:41:43.8737163Z    at org.jose4j.jwk.OctetKeyPairJsonWebKey.<init>(OctetKeyPairJsonWebKey.java:81)
2022-12-31T23:41:43.8738060Z    at org.jose4j.jwk.OctetKeyPairJsonWebKey.<init>(OctetKeyPairJsonWebKey.java:63)
2022-12-31T23:41:43.8738724Z 
2022-12-31T23:41:43.8766974Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisError.parsingError(AnalysisError.java:153)
2022-12-31T23:41:43.8768064Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:104)
2022-12-31T23:41:43.8770213Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.ensureFlowsGraphCreated(MethodTypeFlow.java:83)
2022-12-31T23:41:43.8790663Z ------------------------------------------------------------------------------------------------------------------------
2022-12-31T23:41:43.8801156Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.getOrCreateMethodFlowsGraph(MethodTypeFlow.java:65)
2022-12-31T23:41:43.8801938Z                         11.1s (7.2% of total time) in 26 GCs | Peak RSS: 3.27GB | CPU load: 1.23
2022-12-31T23:41:43.8852201Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.typestate.DefaultVirtualInvokeTypeFlow.onObservedUpdate(DefaultVirtualInvokeTypeFlow.java:109)
2022-12-31T23:41:43.8853143Z ========================================================================================================================
2022-12-31T23:41:43.8853783Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.TypeFlow.update(TypeFlow.java:562)
2022-12-31T23:41:43.8854685Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis$1.run(PointsToAnalysis.java:488)
2022-12-31T23:41:43.8990013Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.executeCommand(CompletionExecutor.java:193)
2022-12-31T23:41:43.8991010Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.lambda$executeService$0(CompletionExecutor.java:177)
2022-12-31T23:41:43.8991823Z 	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1426)
2022-12-31T23:41:43.8992477Z 	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
2022-12-31T23:41:43.8999225Z 	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
2022-12-31T23:41:43.8999951Z 	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
2022-12-31T23:41:43.9000580Z 	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
2022-12-31T23:41:43.9001214Z 	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
2022-12-31T23:41:43.9002456Z Caused by: org.graalvm.compiler.java.BytecodeParser$BytecodeParserError: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: java.security.spec.EdECPoint. This error is reported at image build time because class org.jose4j.keys.EdDsaKeyUtil is registered for linking at image build time by command line
2022-12-31T23:41:43.9008715Z 	at parsing org.jose4j.keys.EdDsaKeyUtil.publicKey(EdDsaKeyUtil.java:81)
2022-12-31T23:41:43.9009444Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.throwParserError(BytecodeParser.java:2518)
2022-12-31T23:41:43.9010429Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.throwParserError(SharedGraphBuilderPhase.java:110)
2022-12-31T23:41:43.9011448Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3393)
2022-12-31T23:41:43.9016948Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.handleBytecodeBlock(BytecodeParser.java:3345)
2022-12-31T23:41:43.9017801Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBlock(BytecodeParser.java:3190)
2022-12-31T23:41:43.9018571Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.build(BytecodeParser.java:1138)
2022-12-31T23:41:43.9019420Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.buildRootMethod(BytecodeParser.java:1030)
2022-12-31T23:41:43.9024789Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.GraphBuilderPhase$Instance.run(GraphBuilderPhase.java:97)
2022-12-31T23:41:43.9025765Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase.run(SharedGraphBuilderPhase.java:84)
2022-12-31T23:41:43.9026529Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.run(Phase.java:49)
2022-12-31T23:41:43.9027355Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:446)
2022-12-31T23:41:43.9032581Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:42)
2022-12-31T23:41:43.9033337Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:38)
2022-12-31T23:41:43.9034113Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.AnalysisParsedGraph.parseBytecode(AnalysisParsedGraph.java:135)
2022-12-31T23:41:43.9035240Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.ensureGraphParsed(AnalysisMethod.java:685)
2022-12-31T23:41:43.9036163Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.parse(MethodTypeFlowBuilder.java:171)
2022-12-31T23:41:43.9041408Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.apply(MethodTypeFlowBuilder.java:349)
2022-12-31T23:41:43.9042334Z 	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:93)
2022-12-31T23:41:43.9042905Z 	... 13 more
2022-12-31T23:41:43.9043784Z Caused by: com.oracle.graal.pointsto.constraints.UnresolvedElementException: Discovered unresolved type during parsing: java.security.spec.EdECPoint. This error is reported at image build time because class org.jose4j.keys.EdDsaKeyUtil is registered for linking at image build time by command line
2022-12-31T23:41:43.9050008Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.reportUnresolvedElement(SharedGraphBuilderPhase.java:333)
2022-12-31T23:41:43.9051143Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.handleUnresolvedType(SharedGraphBuilderPhase.java:288)
2022-12-31T23:41:43.9052238Z 	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.handleUnresolvedNewInstance(SharedGraphBuilderPhase.java:204)
2022-12-31T23:41:43.9053223Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.genNewInstance(BytecodeParser.java:4501)
2022-12-31T23:41:43.9058503Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.genNewInstance(BytecodeParser.java:4494)
2022-12-31T23:41:43.9059399Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBytecode(BytecodeParser.java:5291)
2022-12-31T23:41:43.9060283Z 	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3385)
2022-12-31T23:41:43.9060875Z 	... 28 more
```

### How to Reproduce?

Reproducer:

Steps to reproduce the behavior:

0. make sure you use Windows
1. `git clone https://github.com/quarkus-qe/beefy-scenarios.git`
2. `mvn clean verify -Dnative -Dquarkus.native.native-image-xmx=6g -pl '003-quarkus-many-extensions'`

### Output of `uname -a` or `ver`

Microsoft Windows Server 2022

### Output of `java -version`

Java version: 11.0.17

### GraalVM version (if different from Java)

GraalVM Community, runtime: C:\hostedtoolcache\windows\graalvm-ce-java11-windows\22.3.0\x64\graalvm-ce-java11-22.3.0

### Quarkus version or git rev

999-SNAPSHOT

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.8.6 (84538c9988a25aec085021c365c560670ad80f63)

### Additional information

_No response_

--- a/extensions/smallrye-jwt-build/runtime/src/main/java/io/quarkus/smallrye/jwt/build/runtime/graalvm/Substitutions.java
+++ b/extensions/smallrye-jwt-build/runtime/src/main/java/io/quarkus/smallrye/jwt/build/runtime/graalvm/Substitutions.java
@@ -1,0 +1,29 @@
+package io.quarkus.smallrye.jwt.build.runtime.graalvm;
+
+import java.util.function.BooleanSupplier;
+
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+
+@TargetClass(className = "org.jose4j.jwk.OctetKeyPairJsonWebKey", onlyWith = JavaVersionLessThan17.class)
+final class Target_org_jose4j_jwk_OctetKeyPairJsonWebKey {
+    @Substitute
+    public Target_org_jose4j_jwk_OctetKeyPairJsonWebKey(java.security.PublicKey publicKey) {
+    }
+
+    @Substitute
+    Target_org_jose4j_jwk_OctetKeyPairUtil subtypeKeyUtil() {
+        return null;
+    }
+}
+
+@TargetClass(className = "org.jose4j.keys.OctetKeyPairUtil", onlyWith = JavaVersionLessThan17.class)
+final class Target_org_jose4j_jwk_OctetKeyPairUtil {
+}
+
+class JavaVersionLessThan17 implements BooleanSupplier {
+    @Override
+    public boolean getAsBoolean() {
+        return Runtime.version().version().get(0) < 17;
+    }
+}

